etc/profile-m-z/vmware.profile

```diff
@@ -0,0 +1,36 @@
+# Firejail profile for vmware
+# This file is overwritten after every install/update
+# Description: The industry standard for running multiple operating systems as virtual machines on a single Linux PC.
+include vmware.local
+
+noblacklist ${HOME}/.vmware
+noblacklist ${HOME}/.cache/vmware
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.vmware
+mkdir ${HOME}/.cache/vmware
+whitelist ${HOME}/.vmware
+whitelist ${HOME}/.cache/vmware
+
+include whitelist-common.inc
+
+caps.keep sys_nice
+ipc-namespace
+netfilter
+# dbus-user none
+# dbus-system none
+nogroups
+notv
+shell none
+
+#disable-mnt
```

Comment:
Breaks it something?

Reply:
I haven't noticed anything broken. i have connected pendrive & an android device also. everything works as it should.